### DISCUSSION TITLE
蓝盾并发任务数支持更多

### DISCUSSION THREAD

**Discussion by JeremyJiWZ** [Tencent/bk-ci#4924] [Ideas]
如题，希望可以支持更高并发，如100左右

**Comment by irwinsun** [comment]
你指的是单条流水线的并发？

**Comment by irwinsun** [comment]
目前50的限制是之前统计了现网并发最高的流水线都没有超过40而设置的。

这个限制的设计初衷是：
- 保护整个系统的稳定。
- 避免单条流水线过热，防止无序的滥用，挤占公共资源。


实际可以单独为流水线设置并发数量，产品侧功能未开放。 可以先通过后台为特定流水线调整上限

**Reply by JeremyJiWZ** [reply]
如何调整单条流水线的并发数量？

**Reply by iccengan** [reply]
https://github.com/Tencent/bk-ci/issues/1018
产品功能还没开放哈
